src/main.c

```diff
@@ -25,18 +25,57 @@ int main(int argc, char* argv[])
 		return 1;
 	}
 
-	int result = 0, pos, player = 0;
+	int result = 0, pos, player = 0,i,sw=0;
```

Comment:
Use a more meaningful name instead of `sw` (same for most other newly created variables)